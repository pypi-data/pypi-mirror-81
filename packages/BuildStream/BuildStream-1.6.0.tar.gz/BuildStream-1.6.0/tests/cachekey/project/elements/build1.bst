# The cache key calculation algorithm is the same
# for all build elements, better to only have one test
# for it and without too much noise from changes which
# would cause is to change the test case.
#
# Lets stick with manual, the most basic build element.
kind: manual
sources:
- kind: local
  path: files/local

config:
  build-commands:
  - make
