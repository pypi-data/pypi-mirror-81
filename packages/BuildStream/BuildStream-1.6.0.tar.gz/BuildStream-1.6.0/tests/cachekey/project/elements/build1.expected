36d193b074051c3315afc943cdf4cfb835becac3ea9f18cd1458130bb39e320c